hadoop-tools/hadoop-distcp/src/main/java/org/apache/hadoop/tools/DistCpConstants.java

```diff
@@ -122,8 +122,6 @@ private DistCpConstants() {
   /* DistCp CopyListing class override param */
   public static final String CONF_LABEL_COPY_LISTING_CLASS = "distcp.copy.listing.class";
 
-  public static final String CONF_LABEL_DISTCP_SYNC_CLASS = "distcp.sync.class";
```

Comment:
reinstate. api is tagged as @InterfaceAudience.LimitedPrivate("Distcp support tools"). if it's not supported tag as deprecated, but ideally have it keep working

Reply:
Done. No change to this file.